tests/Main.hs

```diff
@@ -234,11 +234,18 @@ test58b
   -> Proxy (Max (n+2) 1)
 test58b = test58a
 
+test59
+  :: SNat n
+  -> SNat p
+  -> Proxy (1 <=? DivRU (n + 1) (p + 1))
```

Comment:
`Div` requires `KnownNat` constraint. I think GHC introduces this which shouldn't be necssary.